airflow/dags/payments_views/payments_rides.sql

```diff
@@ -153,7 +172,8 @@ SELECT
     t2.latitude AS off_latitude,
     t2.longitude AS off_longitude
 
-FROM `payments.stg_cleaned_micropayments` AS m
+FROM debited_micropayments AS m
+LEFT JOIN refunded_micropayments AS mr USING (micropayment_id)
 JOIN `payments.stg_cleaned_customers` AS c USING (customer_id)
 LEFT JOIN `payments.stg_cleaned_customer_funding_source_vaults` AS v
```

Comment:
nit: is this intended to be an inner join? if so, could it be made explicit?

Reply:
I wanted to make sure these payment fixes got in tonight so I created https://github.com/cal-itp/data-infra/issues/1120 in case we want to work on this later.